CONTRIBUTING.md

```diff
@@ -99,6 +99,17 @@ Fuzzing is heavily encouraged: you will find all related material under `fuzz/`
 
 Mutation testing is work-in-progress; any contribution there would be warmly welcomed.
 
+C/C++ Bindings
+--------------
+
+You can learn more about the C/C++ bindings that are made available by reading the
+[C/C++ Bindings README](lightning-c-bindings/README.md). If you are not using the C/C++ bindings,
+you likely don't need to worry about them, and during their early experimental phase we are not
+requiring that pull requests keep the bindings up to date (and, thus, pass the bindings_check CI
+run). If you wish to ensure your PR passes the bindings generation phase, you should run the
+`genbindings.sh` script in the top of the directory tree to generate, build, and test C bindings on
```

Comment:
Running `genbindings.sh` generates a few files that aren't in the .gitignore. Is it worth adding them:

c-bindings-gen/target/
lightning-c-bindings/target/

Should this end up in target/ or .gitignore?
lightning-c-bindings/include/lightningpp.hpp

Reply:
`target` should definitely be in .gitignore, but at a high level our goal here is to check in the C bindings that are generated so that users don't theoretically need to run the generation scripts (ie can just `cd lightning-c-bindings && cargo test`). CI tests that the versions in git are the same as the ones that are auto-generated.

Reply:
Ok. That makes sense why `lightningpp.hpp` is outside of target. Thanks for the background.

Reply:
Verified the target files are now ignored in 74cd3c9. Thanks!